From 59cd3e17adc0763e25b12848e489941bfa4e9526 Mon Sep 17 00:00:00 2001
From: Neil Cheng <Neil.Cheng@quantatw.com>
Date: Fri, 26 Jun 2026 13:38:47 +0800
Subject: [PATCH] configurations: Add MCTPI2CTarget records for NIC

Add MCTPI2CTarget records for Broadcom 100G and Terminus 100G NICs.

These records allow mctpreactor to derive the physical address encoding
needed to set up the corresponding MCTP endpoints.

Signed-off-by: Neil Cheng <Neil.Cheng@quantatw.com>
---
 configurations/broadcomm/100g_1p_ocp_mezz.json  | 6 ++++++
 configurations/meta/terminus_100g_nic_tsff.json | 6 ++++++
 2 files changed, 12 insertions(+)

diff --git a/configurations/broadcomm/100g_1p_ocp_mezz.json b/configurations/broadcomm/100g_1p_ocp_mezz.json
index d3cb955..c1e040e 100644
--- a/configurations/broadcomm/100g_1p_ocp_mezz.json
+++ b/configurations/broadcomm/100g_1p_ocp_mezz.json
@@ -6,6 +6,12 @@
             "Name": "BRCM OCP NIC FRU $bus",
             "Type": "EEPROM"
         },
+        {
+            "Address": "0x32",
+            "Bus": "$bus",
+            "Name": "NIC",
+            "Type": "MCTPI2CTarget"
+        },
         {
             "Address": "0x1f",
             "Bus": "$bus",
diff --git a/configurations/meta/terminus_100g_nic_tsff.json b/configurations/meta/terminus_100g_nic_tsff.json
index e4f92d7..787d633 100644
--- a/configurations/meta/terminus_100g_nic_tsff.json
+++ b/configurations/meta/terminus_100g_nic_tsff.json
@@ -30,6 +30,12 @@
                 }
             ],
             "Type": "EMC1403"
+        },
+        {
+            "Address": "0x32",
+            "Bus": "$bus",
+            "Name": "NIC",
+            "Type": "MCTPI2CTarget"
         }
     ],
     "Name": "Terminus NIC FRU $bus",
-- 
2.25.1

